### PR TITLE
rename `ruleset` into `assertion`

### LINKED ISSUE #903: rename `ruleset` into `assertion`
### Context
We are going to use the terminology `assertion`, instead of `ruleset`

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

### PR DESCRIPTION
**Pull request by BillyWooo**
minor update

resolves #903

### CODE REVIEW

File: pallets/vc-management/src/assertion.rs [hunk 1]
```diff
@@ -20,26 +20,26 @@
 // See: https://www.notion.so/litentry/Expected-parameters-in-predefined-rulesets-14f74928aa2b43509167da12a3e75507
 
 use codec::{Decode, Encode, MaxEncodedLen};
-use primitives::Balance;
 use scale_info::TypeInfo;
 use sp_runtime::{traits::ConstU32, BoundedVec};
 
+type Balance = u128;
```
Comment (Kailai-Wang) [inline pallets/vc-management/src/assertion.rs]: Any reasons not to use primitives::Balance?

File: runtime/litmus/src/lib.rs [hunk 2]
```diff
@@ -783,6 +784,12 @@ impl pallet_sidechain::Config for Runtime {
 	type EarlyBlockProposalLenience = ConstU64<100>;
 }
 
+impl pallet_teeracle::Config for Runtime {
```
Comment (Kailai-Wang) [inline runtime/litmus/src/lib.rs:786]: Strange, this should be in `dev` already.

**Review by Kailai-Wang — COMMENTED** [review]
Conflict needs to be resolved.

I thought some changes were in `dev` already? Like pallet_teeracle stuff, why does it show up again 😂

File: runtime/litmus/src/lib.rs (hunk 2 above)
Comment (BillyWooo) [reply runtime/litmus/src/lib.rs:786]: Yes. It should be. Git problem.

File: pallets/vc-management/src/assertion.rs (hunk 1 above)
Comment (BillyWooo) [reply pallets/vc-management/src/assertion.rs]: Strange. Didn't noticed this change. Will check.

File: runtime/litmus/src/lib.rs (hunk 2 above)
Comment (Kailai-Wang) [reply runtime/litmus/src/lib.rs:786]: One sec, I just noticed it was from `tee-dev` to `dev`. Maybe we shouldn't do it right now, as these two branches diverge. If you need something badly in the `tee-dev` branch (due to incompatible tee-worker development), it's better to submit a PR to `tee-dev` branch directly.

We will merge `dev` to `tee-dev` once the upstream is updated to v0.9.29

**Comment by Kailai-Wang** [comment]
Could you re-do this PR to only include the naming changes and `docker/litmus-parachain-launch-config.tee-dev.yml`? And from a feature branch other than `tee-dev`?

It looks like merging branches into each other might cause history confusion..

**Comment by BillyWooo** [comment]
close for the moment. Nothing need to update.

**Pull request closed without merging**